### PR TITLE
refactor: use `preconnect` hints for LaunchDarkly domains

### PR DESCRIPTION
**Pull request by emmenko**
Suggestion to improve the connection time and thus rendering performance.

### CODE REVIEW

File: packages/mc-html-template/html-docs/application.html [hunk 1]
```diff
@@ -10,6 +10,11 @@
     <!-- Preconnects -->
     <link rel="preconnect" href="__CDN_URL__">
     <link rel="preconnect" href="__MC_API_URL__">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://app.launchdarkly.com">
+    <link rel="preconnect" href="https://clientstream.launchdarkly.com">
+    <link rel="preconnect" href="https://events.launchdarkly.com">
```
Comment (emmenko) [inline packages/mc-html-template/html-docs/application.html:17]: These three hints are new

File: packages/mc-html-template/src/replace-html-placeholders.ts [hunk 2]
```diff
@@ -15,9 +15,7 @@ const trimTrailingSlash = (value: string) => value.replace(/\/$/, '');
 const getGtmTrackingScript = (gtmId?: string) => {
   if (!gtmId) return '';
   const url = `https://www.googletagmanager.com/gtm.js?id=${gtmId}`;
-  return `
-<script async type="text/javascript" src="${url}" referrerpolicy="no-referrer"></script>
-  `;
+  return `<script async fetchpriority="low" type="text/javascript" src="${url}" referrerpolicy="no-referrer"></script>`;
```
Comment (emmenko) [inline packages/mc-html-template/src/replace-html-placeholders.ts:18]: Adding the `fetchpriority="low"` to loading the GTM script.

**Review by tdeekens — APPROVED** [review]
Good idea!

**Review by CarlosCortizasCT — APPROVED** [review]
💯

**Review by Rhotimee — APPROVED** [review]
🚀

**Pull request merged**